wcivf/apps/people/templates/people/person_detail.html

```diff
@@ -10,356 +10,63 @@
-        </p>
-    {% endif %}
-
-  </div>
-  {% endif %}
-
-{% if object.postelection.husting_set.exists and object.current_or_future_candidacies %}
-    <div class="card info_card">
-      <h3>Local hustings</h3>
-      <p>You can meet candidates and question them at local hustings. Here are hustings where {{ object.name }} may be appearing.</p>
-      {% include "elections/includes/_hustings_list.html" with hustings=object.postelection.husting_set.all %}
-    </div>
-{% endif %}
-
-
-{% if object.past_not_current_candidacies %}
-<div class="card info_card">
-  <h3>Previous elections</h3>
-    <p>This candidate has previously stood for election in:</p>
-    <ul>
-    {% for person_post in object.past_not_current_candidacies %}
-    <li><strong>{{ person_post.election.election_date|date:"Y" }}:</strong>
-      <strong>{{ person_post.election }}</strong>, as the {{ person_post.party.party_name }} candidate for
-      <a href="{{ person_post.post_election.get_absolute_url }}">{{ person_post.post.label }}</a>
-      {{ person_post.post_election.short_cancelled_message_html }}
-    {% endfor %}
-    </ul>
-</div>
-{% endif %}
-
-  {% if referer_postcode %}
-  <div class="back">
-  <p>
-  <a href="{% url 'postcode_view' referer_postcode %}">&laquo; Back to candidates in {{ referer_postcode }}</a>
-  </p>
-  </div>
-  {% endif %}
-
-  {# Edit link #}
-  {% if object.current_or_future_candidacies %}
-  <div class="card" id="findmore">
-  <h3>That's all we know! Will you help us find more about this candidate?</h3>
-
-  <p>Our volunteers have been working hard to add information on as many candidates as possible, but they need help.</p>
-
-  {% if object.should_show_email_cta %}
-  <p>Thousands of voters will rely on this site.{% else %}<p>{% endif %}
-    If you can add information that should be on this page
+      </div>
+      {% endif %}
 
-      {% if object.cta_example_details %}
-      - such as {{ object.name }}'s
-      {{ object.cta_example_details|join:", " }} -
+      {% if not referer_postcode %}
+      {% include "elections/includes/_postcode_search_form.html" %}
       {% endif %}
-      please use our crowdsourcing website to add it.</p>
-    <a href="{{ object.get_ynr_url }}update/" class="link-button">
-      Add or edit details &raquo;
-    </a>
-  {% if object.should_show_email_cta %}
-    <p>You can also email {{ object.name }} directly to ask them to add information to this page.
-  <p><a href="{% url 'email_person_view' pk=object.pk ignored_slug=person.name|slugify %}" class="link-button">Ask the candidate for more information &raquo;</a></p>
-{% endif %}
 
-{#  <h4>Upload your leaflets</h4>#}
-{#  <p>If you've received election leaflets from {{ object.name }}, please take a photo#}
-{#  of them and upload them to ElectionLeaflets.org</p>#}
-{#  <p><a class="link-button" href="https://electionleaflets.org/">Add leaflets</a> </p>#}
+      {#{% include "feedback/feedback_form.html" %}#}
```

Comment:
I reckon we could just delete this and as there has been some talk of changing what sort of feedback we ask for anyway